server/controllers/index.js

```diff
@@ -4,14 +4,12 @@ const { getAllRooms } = require('./getAllRooms')
 const { getAllEvents } = require('./getAllEvents')
 const userEvents = require('./userHome')
 const addEvent = require('../controllers/addEvent')
+const searchAvailableRooms = require('./getAvailableRooms')
 
-router.get('/api/rooms', getAllRooms);
+router.get('/api/rooms', getAllRooms)
 router.get('/api/user-events/:id', userEvents)
 router.get('/api/events', getAllEvents)
+router.get('/api/available-rooms/:date', searchAvailableRooms)
```

Comment:
`date` should be a request parameter, not part of the URL. Pls see https://stackoverflow.com/questions/17007997/how-to-access-the-get-parameters-after-in-express

Reply:
Done